bears/python/PyCheckerBear.py

```diff
@@ -0,0 +1,40 @@
+import shlex
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+        DistributionRequirement)
+
+
+@linter(executable='pychecker',
```

Comment:
`PyChecker` needs to be installed by pip. Add the compatible version of PyChecker in requirements file.